x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_summary/tabs_content/monitor_details_panel.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiBadge,
+  EuiSpacer,
+  EuiLink,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import { capitalize } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { MonitorTags } from './monitor_tags';
+import { MonitorEnabled } from '../../monitors_page/management/monitor_list_table/monitor_enabled';
+import { LocationsStatus } from './locations_status';
+import {
+  getSyntheticsMonitorAction,
+  selectMonitorStatus,
+  syntheticsMonitorSelector,
+} from '../../../state/monitor_summary';
+import { ConfigKey } from '../../../../../../common/runtime_types';
+
+export const MonitorDetailsPanel = () => {
+  const { data } = useSelector(selectMonitorStatus);
+
+  const { monitorId } = useParams<{ monitorId: string }>();
+
+  const dispatch = useDispatch();
+
+  const { data: monitor, loading } = useSelector(syntheticsMonitorSelector);
+
+  if (!data) {
+    return <EuiLoadingSpinner />;
+  }
+
+  return (
+    <>
+      <EuiSpacer />
+      <EuiDescriptionList type="responsiveColumn" style={{ maxWidth: '400px' }}>
```

Comment:
Description list should be smaller. Use `compressed={true}`. Also maxWidth shouldn't be necessary here. If we ever need to define a maxWidth for this, it should be done on the panel at the page layout level.

Reply:
Also if possible, I want to override the default vertical spacing for description list items so that there is an 8px gap rather than 16px (default). This might be the right place to do so. I've spoken to EUI team about adding a vertical spacing prop and it's on my todo list to file a PR for this.